Okta.Xamarin/Okta.Xamarin.iOS/Okta.Xamarin.iOS.csproj

```diff
@@ -99,8 +99,8 @@
     <PackageReference Include="System.Net.Http">
```

Comment:
Should we add the project version here and in the other projects (Xamarin and Android)?

Reply:
Omitting the version should cause it to get the latest, or whatever is available.

Reply:
To add a little more detail, I believe that the System.Net.Http component is wired in a specific way on iOS based on other iOS project settings; the core (or all) of it's functionality is delegated to a native component in iOS (I believe).